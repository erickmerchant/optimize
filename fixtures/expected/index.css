body {
  font: Helvetica;
}

:hover {
  color: blue
}

h1 {
  font-size: 7.59375em;
}

h2 {
  font-size: 5.0625em;
}

h3 {
  font-size: 3.375;
}

h4 {
  font-size: 2.25em;
}

h5 {
  font-size: 1.5em;
}

h6 {
  font-size: 1em;
}

a:hover {
  color: red;
}

@keyframes testing {
  from {
    margin-left: 100%;
  }

  to {
    margin-left: 0%;
  }
}
