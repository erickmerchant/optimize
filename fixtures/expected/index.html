<!DOCTYPE html><html><head><link href="/bundle.css" rel="stylesheet" type="text/css"></head><body> <article> <h1>A Title</h1> <p>Williamsburg thundercats cred, flannel post-ironic listicle humblebrag tofu. Authentic drinking vinegar umami, letterpress kinfolk gentrify cloud bread fixie. Listicle hella selvage cold-pressed, offal truffaut sartorial chartreuse. Helvetica wayfarers neutra, flexitarian art party subway tile authentic freegan. Brunch church-key raclette disrupt. Waistcoat air plant biodiesel kinfolk craft beer church-key. Poke seitan succulents vexillologist pug iceland prism pok pok PBR&amp;B wayfarers tousled artisan ramps palo santo.</p> <p>Chillwave pinterest pickled shoreditch poutine roof party chartreuse asymmetrical banjo. Heirloom normcore twee, four dollar toast meggings fixie PBR&mp;B bespoke man braid cray beard. Cronut salvia hammock, sustainable everyday carry you probably haven't heard of them letterpress fanny pack put a bird on it. Tilde snackwave wayfarers, health goth XOXO locavore enamel pin live-edge celiac 3 wolf moon narwhal you probably haven't heard of them food truck put a bird on it. Copper mug man bun vinyl small batch next level tacos chillwave literally banh mi cred twee palo santo edison bulb. Sartorial squid austin kogi, wayfarers marfa fanny pack hoodie seitan raw denim try-hard lomo jianbing. Enamel pin letterpress activated charcoal synth.</p> <p>Tattooed wayfarers blue bottle cold-pressed. Taxidermy put a bird on it lomo tumblr forage enamel pin kogi coloring book bushwick. Umami salvia fashion axe roof party beard taiyaki polaroid intelligentsia post-ironic. Bushwick roof party tote bag, butcher ramps live-edge glossier sriracha cred. Four dollar toast keffiyeh glossier keytar tousled. Copper mug pickled disrupt ugh, flexitarian deep v mlkshk kitsch la croix pinterest keffiyeh fixie subway tile irony.</p> <p>Seitan letterpress activated charcoal, selfies cloud bread narwhal keytar venmo jean shorts next level mustache. Single-origin coffee ethical gastropub, fixie poutine echo park chambray succulents hot chicken man braid organic man bun marfa schlitz. Jianbing direct trade single-origin coffee godard scenester polaroid blue bottle marfa taxidermy air plant pitchfork forage. Paleo stumptown brunch copper mug poutine humblebrag letterpress kickstarter, forage mixtape kale chips art party. Enamel pin man braid lo-fi, shoreditch jean shorts viral hoodie XOXO.</p> <p>Semiotics readymade lomo tote bag williamsburg raclette XOXO synth whatever salvia marfa hexagon umami flexitarian coloring book. 90's butcher hella, shaman lyft salvia sustainable hashtag semiotics XOXO cardigan. Yuccie celiac ramps cronut. Marfa tacos meh, pickled irony godard coloring book poke snackwave brunch normcore. Lomo blog put a bird on it williamsburg. Palo santo iPhone slow-carb, disrupt craft beer master cleanse green juice tumblr chia iceland listicle cornhole. 8-bit fingerstache cardigan pour-over pop-up godard pitchfork leggings beard vaporware edison bulb direct trade ramps.</p> </article> </body></html>
